### PR TITLE
tests/start.sh: Enable AutoHaT support for Qemu 32 bit images

### PR DESCRIPTION
**Pull request by horia-delicoti**
Connects to resin-os/meta-resin#517
@lifeeth

### CODE REVIEW

**Comment by lifeeth** [comment]
@resin-jenkins test this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ Change log
 
 # v2.0.0-beta.7 - 2016-12-05
 
+* Update tests/start.sh - Enable AutoHaT support for Qemu 32 bit images [Horia]
```
Comment (agherzan) [inline CHANGELOG.md]: This need to be added on top. You can't add something to a release already done.

**Comment by horia-delicoti** [comment]
@resin-jenkins test this please

**Comment by lifeeth** [comment]
Updated the AutoHAT repo as well which is needed to enable the qemu tests.

**Comment by lifeeth** [comment]
The qemu x86 tests dont seem to be functioning properly on jenkins.

**Comment by lifeeth** [comment]
This PR needs to be redone - I have pulled in the new autohat commits in #38

**Comment by horia-delicoti** [comment]
@resin-jenkins test this please

**Comment by horia-delicoti** [comment]
@resin-jenkins ok to test

**Comment by horia-delicoti** [comment]
@resin-jenkins ok to test

**Pull request merged**